microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+1. Click **New InPrivate window**.
 
-   ![The Lighthouse tool](../media/speed-audits-performance.msft.png)
+1. Establish a new baseline in **Lighthouse**:
 
-   <!--todo: add link to Lighthouse when section is available  -->
-   <!-- /web/tools/lighthouse  -->
+   ![The report for the Lighthouse tool about the performance of the site in a InPrivate window](./images/private-lighthouse-report.png)
 
-1. Match your audit configuration settings to those in the previous figure.  Here is an explanation of the different options:
+#### Understand your report
 
-   *  **Device**.  Set to **Mobile** changes the user agent string and simulates a mobile viewport.  Set to **Desktop** pretty much just turns off the **Mobile** changes.
+###### Overall performance score
 
-   *  **Audits**.  Turn off a category to prevent the **Audits** panel from running those audits, and excludes those audits from your report.  Leave the other categories Turned on, if you want to display the types of recommendations that are provided.  Turn off categories to slightly speed up the auditing process.
+The number at the top of your report is the overall performance score for the webpage.  Later, as you make changes to the code, the number displayed should rise.  A higher score means better performance.
 
-   *  **Throttling**.  Set to **Simulated Slow 4G, 4x CPU Slowdown** simulates the typical conditions of browsing on a mobile device.  It is named "simulated" because the **Audits** panel doesn't actually throttle during the auditing process.  Instead, it just extrapolates how long the page takes to load under mobile conditions.  The **Applied...** setting, on the other hand, actually throttles your CPU and network, with the tradeoff of a longer auditing process.
+![The overall performance score](./images/overall-score.png)
 
-   *  **Clear Storage**.  Turn on the checkbox to clear all storage associated with the page before every audit.  Leave this setting on if you want to audit how first-time visitors experience your site.  Turn off this setting when you want the repeat-visit experience.
+###### Metrics
 
-1. Click **Run Audits**.  After 10 to 30 seconds, the **Audits** panel displays a report of the performance of the site.
+The **Metrics** section provides quantitative measurements of the performance of the webpage.  Each metric provides insight into a different aspect of the performance.  For example, **First Contentful Paint** tells you when content first appears on the screen, which is an important milestone in the user's perception of the page load, whereas **Time To Interactive** marks the point at which the page appears ready enough to handle user interactions.
 
-   ![The report for the Audits panel of the performance of the site](../media/speed-glitch-tony-remix-audits-performance-metrics-collapsed.msft.png)
+![The Metrics section](./images/metrics-section.png)
```

Comment:
consider moving up to after sentence "The **Metrics** section provides quantitative measurements of the performance of the webpage:"
and add colon to make that sentence serve as lead-in for image.